aioairzone/zone.py

```diff
@@ -127,6 +129,7 @@ def __init__(self, system_id: int, zone_id: int, zone_data: dict[str, Any]):
         """Zone init."""
         self.air_demand: bool | None = None
         self.anti_freeze: bool | None = None
+        self.aq_status: str | None = None
```

Comment:
According to the [Local API documentation](https://developers.airzonecloud.com/docs/local-api/#get-/hvac) this is supposed to be an `int` instead of a `str`:
```
aq_quality: integer
 Air quality. Read only.

Available devices:

Flexa 3.0 / IBPro 6 / Easyzone, RadianT365, Acuazone / IBPro 32
Aidoo Pro
Available values:

0 Off
1 Good
2 Medium
3 Low
Allowed: 0┃1┃2┃3
```
@didrocks what values do you see on your device?